lib/dialects/postgres/query-generator.js

```diff
@@ -529,11 +529,40 @@ module.exports = (function() {
           template += ' ON UPDATE <%= onUpdateAction %>';
           replacements.onUpdateAction = attribute.onUpdate.toUpperCase();
         }
+
+        if (attribute.references.deferrable) {
+          template += ' <%= deferrable %>';
+          replacements.deferrable = attribute.references.deferrable.toString(this).toUpperCase();
```

Comment:
True, a lot of places just use to uppercase - we haven't been so concerned about this before :)

But in this case, won't be string be uppercase already, if it comes from a deferrable.toSql?